scripts/ops/reclaim_supabase_swap.sql

```
@@ -0,0 +1,104 @@
+-- Capgo-EU Phase A reclaim (run manually in a maintenance window).
+-- REQUIRED: psql (uses \gexec; VACUUM cannot run inside a transaction).
+-- Prefer ~/.pgpass / PGPASSFILE instead of putting the password on the CLI.
+-- Example:
+--   psql "postgresql://postgres@HOST:5432/postgres?sslmode=require" -v ON_ERROR_STOP=1 -f scripts/ops/reclaim_supabase_swap.sql
```

SQLFluff (4.2.2)

[error] Line is too long (128 > 80).

(LT05)